### ISSUE TITLE
Differentiate between area factor and shape factor?

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#1422]
We currently use term shape factor for 2 reason.  In one case we are talking about the cross-sectional shape of a throat (ie. circular vs square) and using it to adjust the hydraulic conductance.  In other cases we are referring to the fact that a pore is a sphere so has a variable cross-sectional area from center to perimeter.  We could call the first an area_factor, while the second is an actual shape factor.  The problem is that shape factor is widely used by other people for the first case, which is how we ended up here.  One option is to use the term volume_factor to refer to the difference between spheres and cubes when calculating effective or average cross-sectional area.

**Comment by ma-sadeghi** [comment]
That's right, and I completely agree. I think this could potentially be linked with issue #1418. It's more convenient to store properties at the bottom most layer of abstraction, i.e. conduit elements (half of pores and throats).

**Comment by TomTranter** [comment]
I disagree (partly) I think that the throat shape factor should remain as shape factor. The pore converging walls could be called a cone-factor or something to convey the main physical reason it exists,

**Comment by jgostick** [comment]
The term area_factor is widely used in the literature to mean the devation of the throat cross-section from a throat.  We used this definition in openpnm but for things other than cross-sectional area, specifically to account for diverging-converging pore-throat-pore conduits.  We have now introduced size_factor models (#1604), which include this effect, as well as the sphere/cone and cuboid/cylinder effects.  We will phase out the use of shape factor in a future release, except for the traditional sense for academic reasons.
